app/views/candidate_interface/english_foreign_language/other_efl_qualification/_form_fields.html.erb

```diff
@@ -14,7 +14,6 @@
   :award_year,
   label: { text: 'When did you complete the assessment?', size: 'm' },
   hint: { text: 'Give the year, for example: ‘2020’' },
-  type: :number,
```

Comment:
This is still useful — or is it now automatically added by the formbuilder?

Reply:
I removed because it was causing an error where an '.' and/or 'e' input would actually send no value in the params causing it trigger the blank validation and not retain the value. Happy to put it back in, but that edge case will remain

Reply:
form now allows letters but they will get caught by validation and also retain the value

Reply:
Yeah, we should avoid `<input type=number>` as it has loads of browser bugs unfortunately. Instead can you add ` pattern="[0-9]*" inputmode="numeric"` to the input? This should then trigger the numeric keypad on mobile devices.

(This is what the [date component](https://design-system.service.gov.uk/components/date-input/) does)